cmd/fyne_demo/screens/window.go

```diff
@@ -161,6 +171,20 @@ func loadDialogGroup(win fyne.Window) *widget.Card {
 				fileSaved(writer)
 			}, win)
 		}),
+		widget.NewButton("Color Picker", func() {
+			picker := dialog.NewColorPicker("Pick a Color", "What is your favorite color?", func(c color.Color) {
+				colorPicked(c, win)
+			}, win)
+			picker.Show()
+		}),
+		widget.NewButton("Advanced Color Picker", func() {
+			picker := dialog.NewColorPicker("Pick a Color", "What is your favorite color?", func(c color.Color) {
+				colorPicked(c, win)
+			}, win)
+			picker.Advanced = true
+			picker.Refresh()
```

Comment:
Can you take this line out so we know it works without :)

Reply:
Done!